solr/core/src/test/org/apache/solr/handler/TestReplicationHandler.java

```diff
@@ -87,18 +85,18 @@
  *
  * @since 1.4
  */
-@LuceneTestCase.Nightly
+// @LuceneTestCase.Nightly
```

Comment:
These changes shouldn't be here?

Reply:
correct..   I need to figure out how to run a nightly unit test from inside of intellij..  i keep commenting this out to run the tests.....

Reply:
also, on my slow laptop it takes 110 seconds roughly to run this test..   what is the criteria for being a nightly?